# How many motors does robot have ?
---
# Robot initial state
float64 q
float64 dq
float64 tau